contributed_definitions/nyaml/NXraman.yaml

```
@@ -0,0 +1,755 @@
+    direction is varied.
+  N_observables: |
+    Number of observables that are saved in a measurement. e.g. one for
+    intensity, reflectivity or transmittance, two for Psi and Delta etc. This
+    is equal to the second dimension of the data array 'measured_data' and the
+    number of column names.
+  N_time: |
+    Number of time points measured, the length of NXsample/time_points
+  N_spectrum_filter: |
+    Number of points for a spectral filter used in the setup
+
+# 03/2024
+# A draft version of a NeXus application definition for Raman spectroscopy.
+
+# The document has the following main elements:
+# - Instrument used and is characteristics
+# - Sample: Properties of the sample
+# - Data: measured data, data errors
+# - Derived parameters: e.g. extra parameters derived in the measurement software
+type: group
+NXraman(NXopt):
+  (NXentry):
+    doc: |
+      This is the application definition describing Raman spectroscopy experiments.
+      
+      Such experiments may be as simple a single Raman spectrum from spontanous
+      Raman scattering and range to Raman imaging Raman spectrometer,
+      surface- and tip-enhanced Raman techniques, x-Ray Raman scattering, as well
+      as resonant Raman scattering phenomena or multidimenional Raman spectra (i.e.
+      varying temperature, pressure, electric field, ....)
+      
+      The application definition defines:
+      
+      * elements of the experimental instrument
+      * calibration information if available
+      * parameters used to tune the state of the sample
+      * sample description
+    definition:
+      doc: |
+        An application definition for ellipsometry.
+      \@version:
+        doc: |
+          Version number to identify which definition of this application
+          definition was used for this entry/data.
+      \@url:
+        doc: |
+          URL where to find further material (documentation, examples) relevant
+          to the application definition.
+      enumeration: [NXraman]
+    experiment_description:
+      doc: |
+        An optional free-text description of the experiment.
+        
+        However, details of the experiment should be defined in the specific
+        fields of this application definition rather than in this experiment
+        description.
+    experiment_type:
+      doc: |
+        Specify the type of Raman measurement.
+      enumeration: [in situ Raman spectroscopy, resonant Raman spectroscopy, non-resonant Raman spectroscopy, Raman imaging, Tip-enhanced Raman spectroscopy (TERS), Surface-enhanced Raman spectroscopy (SERS), Surface plasmon polariton enhanced Raman scattering (SPPERS), Hyper Raman spectroscopy (HRS), Stimulated Raman spectroscopy (SRS), Inverse Raman spectroscopy (IRS), Coherent anti-Stokes Raman spectroscopy (CARS)]
+    (NXinstrument):
+      doc: |
+        Properties of the Instruments used for Raman instrumeasurements.
```

```suggestion
        Properties of the instruments used for Raman measurements.
```

Maybe not properties but rather metadata or so?

---

Ive addressed this